waku/node/v2/wakunode2.nim

```diff
@@ -125,20 +128,21 @@ proc start*(node: WakuNode) {.async.} =
   node.switch.mount(node.wakuFilter)
   node.subscriptions.subscribe(WakuFilterCodec, node.wakuFilter.subscription())
 
+proc mountStore*(node: WakuNode) =
+  info "mounting store"
+  node.wakuStore = WakuStore.init(node.switch, node.rng)
+  node.switch.mount(node.wakuStore)
+  node.subscriptions.subscribe(WakuStoreCodec, node.wakuStore.subscription())
+
+proc mountRelay*(node: WakuNode) {.async, gcsafe.} =
```

Comment:
In mountRelay nothing gets mounted, as this happens in `init` already. 
Why this difference with the other protocols exactly? Could it be made consistent? Or would there be a downside to that?

Or could all get mounted in `init`, and then just run subscriptions/subscribe per protocol to activate it?
What would that do on a libp2p level? How would the connections look at after that `init` state then?